CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/visual/DefaultInteractionEventHandler.java

```diff
@@ -143,7 +146,7 @@ public class DefaultInteractionEventHandler implements InteractionEventHandler {
 
     private final SelectionFreeformModel freeformModel = new SelectionFreeformModel();
 
-    private boolean announceNextFlush = false;
+    private final boolean announceNextFlush = false;
```

Comment:
would there be any reason why we would want to make this true? If so, we should probably remove the final and make it true somewhere. If not, we probably don't need to have a separate variable to hold a boolean constant

Reply:
Removed this variable